scripts/cwd.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+current_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $current_dir/utils.sh
```

Comment:
```
source $current_dir/utils.sh
^-------------------^ SC1091 (info): Not following: ./utils.sh was not specified as input (see shellcheck -x).
^----------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```